processor/src/main/java/com/linecorp/decaton/processor/runtime/internal/AbstractSubPartitions.java

```diff
@@ -59,7 +59,7 @@ protected AbstractSubPartitions(PartitionScope scope, Processors<?> processors)
         rateLimiter = new DynamicRateLimiter(rateProperty(scope));
         Metrics metrics = Metrics.withTags(
                 "subscription", scope.subscriptionId(),
-                "topic", scope.topic(),
+                "topic", scope.topicPartition().topic(),
```

Comment:
in addition to these changes I wanna make the following changes to ensure we won't repeat the same mistake in future. Are you happy to follow these up in this PR? Otherwise I'll cover that after merging this PR.

* Let's use variable to assign `scope.topicPartition()` return and use it instead to avoid unnecessary object allocation.
* Rename `SubscriptionScope.topic` to be `SubscriptionScope.originTopic` to distinguish it w/ PartitionScope.topicPartition

Reply:
Thank you for your review. I see, I'll update them as well.